OWNERS_ALIASES

```diff
@@ -0,0 +1,30 @@
+aliases:
+  tech-leads:
+    - csrwng
+    - enxebre
+    - sjenning
+  core-approvers:
+    - bryan-cox
+    - csrwng
+    - enxebre
+    - jparrill
+    - muraee
+    - sjenning
+  core-reviewers:
+    - bryan-cox
+    - csrwng
+    - devguyio
+    - enxebre
+    - jparrill
+    - muraee
+    - sdminonne
+    - sjenning
+  kubevirt-approvers:
+    - davidvossel
```

Comment:
I don't think David should be included anymore. I thought he left kubevirt. I think Oren should be added perhaps?

Same comment for reviewers below.